tests/core/test_solve.py

```diff
@@ -78,6 +81,10 @@ def test_solve_1(tmpdir):
 
 
 def test_solve_2(tmpdir):
+    if context.solver == "libmamba":
+        pytest.skip("conda-libmamba-solver does not use the SSC object")
```

Comment:
SSC?

Reply:
`SolverStateContainer`. It's part of the `Resolve` (`r`) object that the (`Solver`) instance embeds; aka `Solver(...).r._ssc`.